lib/ui/modals/search_modal.dart

```diff
@@ -18,7 +22,11 @@ class SearchBottomModal extends StatelessWidget {
             top: context.responsive(14),
             bottom: context.responsive(14) + viewInsets + safeAreaBottom,
           ),
-          child: SearchBar(margin: EdgeInsets.zero),
+          child: SearchBar(margin: EdgeInsets.zero, onSubmit: (content) {
+              scheduleMicrotask(() {
```

Comment:
What is `scheduleMicrotask` doing in this case?

Reply:
Oups, that was a copy paste mistake sorry 😅